it/src/test/java/apoc/it/core/WarmupEnterpriseTest.java

```diff
@@ -20,51 +20,81 @@
 
 import static apoc.util.TestContainerUtil.createEnterpriseDB;
 import static apoc.util.TestContainerUtil.testCall;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotEquals;
 import static org.junit.Assert.assertThrows;
 import static org.junit.Assert.assertTrue;
 
-import apoc.util.Neo4jContainerExtension;
 import apoc.util.TestContainerUtil;
 import java.util.List;
+import java.util.Map;
+import java.util.function.Consumer;
 import org.junit.Test;
 import org.neo4j.configuration.GraphDatabaseInternalSettings;
 import org.neo4j.configuration.GraphDatabaseSettings;
+import org.neo4j.driver.Driver;
 import org.neo4j.driver.Session;
+import org.neo4j.driver.SessionConfig;
+import org.neo4j.driver.exceptions.ClientException;
 
 public class WarmupEnterpriseTest {
 
     @Test
     public void testWarmupIsntAllowedWithOtherStorageEngines() {
-        Neo4jContainerExtension neo4jContainer = createEnterpriseDB(List.of(TestContainerUtil.ApocPackage.CORE), true)
-                .withNeo4jConfig(GraphDatabaseInternalSettings.include_versions_under_development.name(), "true")
-                .withNeo4jConfig(GraphDatabaseSettings.db_format.name(), "multiversion");
-        neo4jContainer.start();
-        Session session = neo4jContainer.getSession();
-
-        RuntimeException e =
-                assertThrows(RuntimeException.class, () -> testCall(session, "CALL apoc.warmup.run()", (r) -> {}));
-        assertTrue(e.getMessage().contains("Record engine type unsupported"));
-
-        session.close();
-        neo4jContainer.close();
+        final var conf = Map.of(
+                GraphDatabaseInternalSettings.include_versions_under_development.name(), "true",
+                GraphDatabaseSettings.db_format.name(), "multiversion");
+        withSession(conf, session -> {
+            RuntimeException e =
+                    assertThrows(RuntimeException.class, () -> testCall(session, "CALL apoc.warmup.run()", (r) -> {}));
+            assertTrue(e.getMessage().contains("Record engine type unsupported"));
```

Comment:
Didn't you delete this message? 🤔

Reply:
Nice catch! I did, I assume this test is not run as part of the github build? 🤔🤔🤔

Reply:
fbf1a8a02baccb87a58c76a621b9d47f0aab3986